ci-operator/step-registry/baremetalds/assisted/operator/gather/baremetalds-assisted-operator-gather-ref.yaml

```diff
@@ -1,6 +1,6 @@
 ref:
   as: baremetalds-assisted-operator-gather
-  from: assisted-service-scripts
```

Comment:
can you remove generation of ``assisted-service-scripts`` image on the branch configuration?

Reply:
There is another dependency to change for `assisted-service-scripts` -- can we hold on this part?

Reply:
sure thing
I wasn't aware of publish step using it, so thought we don't need that anymore